include/igl/opengl/MeshGL.cpp

```diff
@@ -46,6 +48,26 @@ IGL_INLINE void igl::opengl::MeshGL::init_buffers()
   glGenBuffers(1, &vbo_points_V);
   glGenBuffers(1, &vbo_points_V_colors);
 
+  // Text Labels
+  glGenVertexArrays(1, &vertex_labels.vao_labels);
```

Comment:
The `TextGL` class could have methods `init_buffers()` and `free_buffers()` to avoid repetitions here.